webpack/InsightsVulnerability/InsightsVulnerability.js

```diff
@@ -1,13 +1,45 @@
-import React from 'react';
+import React, { useEffect } from 'react';
 import PageLayout from 'foremanReact/routes/common/PageLayout/PageLayout';
 import { translate as __ } from 'foremanReact/common/I18n';
+import { ScalprumComponent } from '@scalprum/react-core';
+import { ScalprumContext, ScalprumContextWrapper } from '../common/ScalprumModule/ScalprumContext';
 
-const InsightsVulnerability = () => (
-  <PageLayout searchable={false} header={__('Vulnerability')}>
-    <div className="insights-vulnerability">
-      <p>This page is under development. Please check back soon for updates.</p>
+const InsightsVulnerability = () => {
+  const { config, setConfig } = React.useContext(ScalprumContext);
+  const cdnPath = 'apps/landing'
+  const scope = 'landing'
+  const module = './AnsibleWidget'
```

Comment:
The module should be `./CveListPage`.

Reply:
Thanks! I’ve updated everything. This component was only used for testing purposes, as the CveListPage isn't functional in this code yet.